apps/src/StudioApp.js

```diff
@@ -809,7 +809,7 @@ StudioApp.prototype.renderShareFooter_ = function (container) {
       },
       {
         text: window.dashboard.i18n.t('footer.how_it_works'),
-        link: location.href + "/edit",
+        link: location.href + "/view",
```

Comment:
I think this is also used for the "Built on Code Studio" menu on legacy shares, e.g. https://studio.code.org/c/200000000.  For that page /edit is valid but /view is 404.